spec/diddocuments.md

```diff
@@ -713,9 +706,11 @@ Resulting DID document:
 ```json
 {
   "id": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+  "didDocumentMetadata": {
```

Comment:
This structure is not correct. The DID document metadata is outside the DID document, see here: https://w3c-ccg.github.io/did-resolution/#did-resolution-result

So it should be like this:

**Resulting DID Resolution Result** (includes DID document and metadata):

```json
{
  "didDocument": { ... },
  "didDocumentMetadata": { ... },
  "didResolutionMetadata": { ... }
}
```

Reply:
Good catch. I'll be updating the examples with actual output from the actual getting started example soon.

Reply:
I have submitted a fix so we can push this into the public review